compiler/noirc_evaluator/src/ssa/interpreter/tests/mod.rs

```diff
@@ -373,6 +382,309 @@ fn store_with_aliases() {
     executes_with_no_errors(src);
 }
 
+#[test]
+fn test_msm() {
```

Comment:
Can you move these tests to either the `instructions.rs` tests or a new file for intrinsics/blockbox tests?

Reply:
Done